mitodl/ocw-hugo-themes
# Resource page template

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to https://github.com/mitodl/ocw-studio/issues/508
Fixes #173 

#### What's this PR do?
Updates the template to handle resource pages which will be produced by https://github.com/mitodl/ocw-to-hugo/pull/337. The same resource template will be used for PDFs, youtube videos, and other file types. We will handle images in a future PR #184 

#### How should this be manually tested?
Convert `18-06-linear-algebra-spring-2010` using https://github.com/mitodl/ocw-to-hugo/pull/337 and view it using this PR. In general everything should look the same, except:
 - Inline images which used to link to resources may not show up until this is addressed https://github.com/mitodl/ocw-to-hugo/issues/352
 - There is a resource page for downloading files. I don't know if there are any links to them at the moment but you can go to http://localhost:3000/resources/18-06_l05/ for example. The download link should work and it should look similar to the mockup: https://projects.invisionapp.com/d/main?origin=v7#/console/18609637/451119818/inspect?scrollOffset=742
 - Video resources should look just about the same as before. The link in this PR is `resources/lecture-2-elimination-with-matrices/` and the corresponding page on production is [here](https://ocwnext.odl.mit.edu/courses/18-06-linear-algebra-spring-2010/pages/video-lectures/lecture-2-elimination-with-matrices/).  I noticed a shift in padding but no other changes.
 - Click on the assignments link and click a PDF link. It should have a URL like `/resources/mit18_06s10_pset7_s10_soln/` and render the expected PDF. The corresponding page is [here](https://ocwnext.odl.mit.edu/courses/18-06-linear-algebra-spring-2010/pages/assignments/mit18_06s10_pset7_s10_soln/) on production

#### Screenshots
![Screenshot from 2021-09-16 11-23-14](https://user-images.githubusercontent.com/863262/133639983-9b9c0eb5-4efe-49ce-b6dd-ee395f5c2f88.png)
![Screenshot from 2021-09-16 11-29-58](https://user-images.githubusercontent.com/863262/133640976-4d2f2b2a-b1bc-41c7-b9ad-f7345fd44223.png)
![Screenshot from 2021-09-16 11-33-55](https://user-images.githubusercontent.com/863262/133641612-6ce70cc1-714a-45ab-9646-47b45595922b.png)
